ibm/service/scc/resource_ibm_scc_scope.go

```diff
@@ -0,0 +1,735 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package scc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/scc-go-sdk/v5/securityandcompliancecenterapiv3"
+)
+
+func ResourceIbmSccScope() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMSccScopeCreate,
+		ReadContext:   resourceIBMSccScopeRead,
+		UpdateContext: resourceIBMSccScopeUpdate,
+		DeleteContext: resourceIBMSccScopeDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"instance_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_scc_scope", "instance_id"),
+				Description:  "The ID of the Security and Compliance Center instance.",
+			},
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_scc_scope", "name"),
+				Description:  "The scope name.",
+			},
+			"description": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_scc_scope", "description"),
+				Description:  "The scope description.",
+			},
+			"environment": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_scc_scope", "environment"),
+				Description:  "The scope environment. This value details what cloud provider the scope targets.",
+			},
+			// Manual Change: change name and value for scope_type and scope_id
+			"properties": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "A list of scopes/targets to exclude from a scope.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id": &schema.Schema{
```

Comment:
Allowed properties are 
scope_id
scope_type
exclusions
provider_type
provider_type_instance_id

Let's use only these and no where should we reference ibm_facts_api_instance_id